docs/setting-up/client/mongodb.md

```diff
@@ -33,140 +64,213 @@ db.getSiblingDB("admin").createRole({
+**Example**
 
 ```sh
-pmm-admin add mongodb --username=pmm --password=pmm --service-name=mongo --host=127.0.0.1 --port=27017
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+--service-name=mymongosvc --host=127.0.0.1 --port=27017
 ```
 
-You can add a MongoDB instance using a UNIX socket with the `--socket` option:
+**Example -- connect via UNIX socket**
 
 ```sh
 pmm-admin add mongodb --socket=/tmp/mongodb-27017.sock
 ```
 
-> If the password contains special symbols like the 'at' (`@`) symbol, the host might not be detected correctly. Make sure that you insert the password with special characters replaced with their escape sequences. The simplest way is to use the [`encodeURIComponent`][ENCODE_URI] JavaScript function in your browser's web console (usually found under *Development Tools*). Evaluate the function with your password as the parameter. For example:
->
-> ```javascript
-> encodeURIComponent('$ecRet_pas$w@rd')
-> "%24ecRet_pas%24w%40rd"
-> ```
 
-[ENCODE_URI]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
+**Example -- connecting via SSL/TLS**
+
+```sh
+pmm-admin add mongodb --tls \
+--tls-certificate-key-file=PATHTOCER \
+--tls-certificate-key-file-password=IFPASSWORDTOCERTISSET \
+--tls-ca-file=PATHTOCACERT
+```
+
+where:
+
+- `PATHTOCERT`: Path to TLS certificate file.
+- `IFPASSWORDTOCERTISSET`: Password for TLS certificate file.
+- `PATHTOCACERT`: Path to certificate authority file.
+
+> **See also** [`pmm-admin` man page for `pmm-admin add mongodb`](../../details/commands/pmm-admin.md#mongodb)
+
+## Tips
+
+- When adding nodes of a replica set, add each node separately using the same cluster name.
 
-## Passing SSL parameters to the MongoDB monitoring service
+- If the password contains special symbols (such as `@`), the host might not be detected correctly. You should convert any password with special characters, replacing them with their escape sequence equivalents.
 
-SSL/TLS related parameters are passed to an SSL enabled MongoDB server as
-monitoring service parameters along with the `pmm-admin add` command when adding
-the MongoDB monitoring service.
+    One way to do this is to use the [`encodeURIComponent`][ENCODE_URI] JavaScript function in your browser's web console (usually found under *Development Tools*). Evaluate the function with your password as the parameter. For example:
 
-Run this command as root or by using the `sudo` command
+    ```js
+    > encodeURIComponent("s3cR#tpa$$worD")
+    ```
+
+    will give:
+
+    ```
+    "s3cR%23tpa%24%24worD"
+    ```
+
+## Check the service
```

Comment:
Is this a troubleshooting tip?   There's a heading Tip above but this appears to be a new section and I'm not sure why it's standalone as it feels like something I need to do but it seems optional for troubleshooting purposes.

Reply:
It's a hangover from before reworking. Yes, it seems out of place as it's a general tip that could apply to all passwords. (It's a Romaism.) I'll move it to Troubleshooting and link to it from here.